### PR TITLE
add categories page of quiz

### PR DESCRIPTION
**Pull request by Ayushi20-19**
(navbar code is extra as the home page is not merged i will remove that later and import homepage nav style)
(also UI colors are random i will refactor them later)
Screenshot (692)

### CODE REVIEW

File: categories.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            <span> <a href="./index.html">Home</a> </span>
+            <li><a href="#">Categories</a></li>
+            <li><a href="#">ScoreCards</a></li>
+            <li><a href="#">Login/Signup</a></li>
+          </sidebar>
+        </div>
+        <div class="categories-content">
+          <div class="genre-card">
+            <div class="image-genre">
+              <img
```
Comment (prakash855) [inline categories.html:45]: Add alt to all your img elements

Comment (Ayushi20-19) [reply categories.html:45]: okay

**Review by prakash855 — APPROVED** [review]
Add alt to all your img elements, rest is fine

File: categories.css [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  --home-primary-color: #6500b3;
+  --home-secondary-color: #d392ff;
+  --home-contast-color: #2dffd9;
+  --black-color: #000000;
+  --white-color: #ffffff;
+}
+
+main {
+  width: 100%;
+  /* padding: 0 5%; */
```
Comment (Divya70) [inline categories.css:16]: remove unnecessary code

File: categories.css [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+  top: 0;
+  padding-top: 5rem;
+  position: absolute;
+  left: -200rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  transition: all 1s;
+  z-index: 2;
+  /* background-color: var(--home-primary-color); */
```
Comment (Divya70) [inline categories.css:77]: here also

**Review by Divya70 — APPROVED** [review]

**Pull request merged**